src/admin.rs

```diff
@@ -69,6 +70,7 @@ enum Command {
     FactoryReset,
     #[cfg(feature = "factory-reset")]
     FactoryResetApp,
+    ListAvailableConfig,
```

Comment:
nit: I would just call it `ListConfig`/`LIST_CONFIG` to be less verbose, but that’s just a personal preference

Reply:
Intuitively, `ListConfig` would also return the values for each config field.